docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,327 @@
+                    "ecr:PutImage"
+                ],
+                "Resource": [
+                    "arn:aws:ecr:*:*:repository/python36",
+                    "arn:aws:ecr:*:*:repository/tf",
+                    "arn:aws:ecr:*:*:repository/filebeat",
+                    "arn:aws:ecr:*:*:repository/sklearn",
+                    "arn:aws:ecr:*:*:repository/*/python36",
+                    "arn:aws:ecr:*:*:repository/*/tf",
+                    "arn:aws:ecr:*:*:repository/*/filebeat",
+                    "arn:aws:ecr:*:*:repository/*/sklearn"
+                ]
+            },
+            {
+                "Sid": "AllowGetAuthToken",
+                "Effect": "Allow",
+                "Action": "ecr:GetAuthorizationToken",
+                "Resource": "*"
+            },
+            {
+                "Sid": "AllowDescirbeEKS",
+                "Effect": "Allow",
+                "Action": "eks:DescribeCluster",
+                "Resource": "arn:aws:eks:*:*:cluster/*"
+            },
+            {
+                "Sid": "AllowFullAccessToS3Bucket",
+                "Effect": "Allow",
+                "Action": "s3:*",
+                "Resource": "*"
+            }
+        ]
+    }
+
+
+Step 3: Allow your role to use your EKS cluster 
+-----------------------------------------------
+You need to give your role permissions to access your EKS cluster using the following kubectl command. For more details, check `Managing users or IAM roles for your cluster
+<https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html>`_. The kubectl edit command uses *vi* editor by default, however, you can override this behaviour by setting *KUBE_EDITOR* to your preferred editor, check `Kubernetes editing resources <https://kubernetes.io/docs/reference/kubectl/cheatsheet/#editing-resources>`_. 
+
+.. code-block:: bash 
+    
+    KUBE_EDITOR="vi" kubectl edit configmap aws-auth -n kube-system
+
+Output:
+
+.. code-block:: bash 
+
+    # Please edit the object below. Lines beginning with a '#' will be ignored,
+    # and an empty file will abort the edit. If an error occurs while saving this file will be
+    # reopened with the relevant failures.
+    #
+    apiVersion: v1
+    data:
+    mapRoles: |
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: arn:aws:iam::755182613526:role/eksctl-my-eks-cluster-nodegroup-m-NodeInstanceRole-FQ7L0HQI4NCC
+          username: system:node:{{EC2PrivateDNSName}}
+    kind: ConfigMap
+    metadata:
+    creationTimestamp: "2020-08-24T07:42:31Z"
+    name: aws-auth
+    namespace: kube-system
+    resourceVersion: "770"
+    selfLink: /api/v1/namespaces/kube-system/configmaps/aws-auth
+    uid: c794b2d8-9f10-443d-9072-c65d0f2eb552
+
+Follow the example below (lines 13-16) to add your role to *mapRoles* and assign *system:masters* group to your role. Make sure to replace 'YOUR ROLE RoleARN' with your role RoleARN before saving.
```

Comment:
In the example below it's `YOUR INSTANCE PROFILE ARN` Also, can we make it more bold that it's the `Instance Profile ARN` and not the `Role ARN`

Reply:
Ignore the comment above.

Just make the note below `Warning` I missed it, that's why I wrote the previous comment ;)